storage/google/cloud/storage/blob.py

```diff
@@ -30,6 +30,9 @@
 import six
 from six.moves.urllib.parse import quote
 
+import google.auth.transport.requests as tr_requests
```

Comment:
I **try** to stick to the [Google Python style guide][1]:

> Use imports for packages and modules only.

[1]: https://google.github.io/styleguide/pyguide.html#Imports